src/main/java/org/project_kessel/api/relations/v1beta1/CheckRequestValidator.java

```diff
@@ -0,0 +1,46 @@
+// Code generated by protoc-gen-validate. DO NOT EDIT.
+// source: kessel/relations/v1beta1/check.proto
+
+package org.project_kessel.api.relations.v1beta1;
```

Comment:
Hmm... this approach brings the generated classes back into the codebase, which I'm not sure is optimal. It's more typical in java for these generated classes to not be committed to SCM and be generated in target and put into jars at build time instead.

It might be fine, just worth some thought.

Reply:
Yup, can be removed and just use imports `build.buf.gen.kessel.relations.v1beta1` and `build.buf.gen.kessel.relations.v1`